src/code_memory/server.py

```
@@ -0,0 +1,199 @@
+"""MCP server implementation for Code Memory using FastMCP framework."""
+
+import logging
+import os
+from collections.abc import AsyncIterator
+from contextlib import asynccontextmanager
+from dataclasses import dataclass
+from typing import List, Dict
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)